xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/html_displayer/entitytreenode/edit.vm

```diff
@@ -0,0 +1,84 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+#if ($displayer.mode == 'edit')
+#set ($entityTypeMapping = {
+  'farm': 'CUSTOM',
+  'wiki': 'CUSTOM',
+  'space': 'DOCUMENT',
+  'document': 'DOCUMENT',
+  'attachments': 'DOCUMENT',
+  'attachment': 'ATTACHMENT',
+  'translations': 'DOCUMENT',
+  'objects': 'DOCUMENT',
+  'objectsOfType': 'CUSTOM'
+})
+## We define a container span to identify the elements to synchronize.
+<span class="entitytreenode-container">
+## Main input is hidden since it's serialization of the combination of the displayed fields.
+## That's also why we need the javascript to keep this hidden input in sync.
+#set ($discard = $xwiki.jsfx.use('uicomponents/suggest/entitytreenode.js', true))
+#set ($discard = $xwiki.ssfx.use('uicomponents/suggest/entitytreenode.css', true))
```

Comment:
```suggestion
#set ($discard = $xwiki.jsfx.use('uicomponents/suggest/entitytreenode.js'))
#set ($discard = $xwiki.ssfx.use('uicomponents/suggest/entitytreenode.css'))
```

I don't see any Velocity code being used in the JavaScript or CSS code.

Reply:
Right, I wrongly thought it was related to minify...